cmd/contour/ingressstatus.go

```diff
@@ -108,24 +109,30 @@ func (isw *loadBalancerStatusWriter) Start(stop <-chan struct{}) error {
 
 func parseStatusFlag(status string) v1.LoadBalancerStatus {
 
-	// Use the parseability by net.ParseIP as a signal, since we need
-	// to pass a string into the v1.LoadBalancerIngress anyway.
-	if ip := net.ParseIP(status); ip != nil {
-		return v1.LoadBalancerStatus{
-			Ingress: []v1.LoadBalancerIngress{
-				{
-					IP: status,
-				},
-			},
+	// Support ','-separated lists
+	items := strings.Split(status, ",")
+	ingresses := make([]v1.LoadBalancerIngress, len(items))
+	count := 0
+
+	for _, item := range items {
```

Comment:
I think that we can drop the preallocation and simplify. Then `append` the items and don't worry about tracking the count.

```suggestion
	ingresses := []v1.LoadBalancerIngress{}
	
	for _, item := range strings.Split(status, ",") {
```

Reply:
I'd automatically written it that way to avoid (theoretical) extra memory allocations. But of course, the chances are that the number of items is small anyway, in which case it hardly matters and you're right, it seems simpler not to.

Done.